net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
It can be reworked to use only uci-defaults though.  They will be sourced in `default_postinst` on either first boot or pkg installation happened later.  This way we can dump the postinst section altogether.